packages/components/select/src/select.stories.ts

```diff
@@ -16,42 +19,149 @@ const onSubmit = (event: Event & { target: HTMLFormElement }): void => {
   data.forEach((value, key) => (output.textContent += `${key}: ${value.toString()}\n`));
 };
 
+const states: string[] = ['', 'valid', 'invalid'];
+const disabledStates = [false, true];
+const sizes: SelectSize[] = ['md', 'lg'];
+
 export default {
   title: 'Select',
   argTypes: {
     maxOverlayHeight: {
       control: 'text'
+    },
+    size: {
+      control: 'inline-radio',
+      options: sizes
     }
   }
 };
 
-export const API: StoryObj = {
+export const Basic: StoryObj = {
   args: {
+    size: 'md',
     maxOverlayHeight: '200px'
   },
-  render: ({ maxOverlayHeight }) => html`
+  render: ({ maxOverlayHeight, size }) => html`
+    <style>
+      sl-select {
+        width: 400px;
+        max-width: 90vw;
+        display: inline-flex;
+      }
+    </style>
     <sl-button>To focus</sl-button>
-    <sl-select maxOverlayHeight=${maxOverlayHeight}>
+    <sl-select maxOverlayHeight=${maxOverlayHeight} .size="${size}">
```

Comment:
what is `maxOverlayHeight`? There is no dot in front of it, so it isn't a property? And attributes names should be snake-case